src/ocm/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -160,7 +165,7 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
             <ToolbarButton
               id="cluster-installation-logs-button"
               variant={ButtonVariant.link}
-              onClick={() => downloadClusterInstallationLogs(addAlert, cluster.id)}
+              onClick={() => void downloadClusterInstallationLogs(addAlert, cluster.id)}
```

Comment:
Same here, we can fix this issues in the future. It's in multiple places, so it's safer to revert all of them and test each case individualy.

Reply:
reverting here